less/home-page.less

```diff
@@ -47,15 +68,49 @@
     }
   }
 
+.the-bus {
+  width: 800px;
+  margin: 0 auto;
+  border-bottom: 2px dashed #C0C0C0;
+}
+.header {
+  padding-top: 20px;
+  border-bottom: 2px dashed #C0C0C0;
+}
+.header p {
+  padding-bottom: 20px;
+}
+
   .content-pick {
     padding-top: 30px;
     border-top: 2px dashed @navigation-border;
     display: flex;
     justify-content: space-between;
+    @media(max-width: 500px){
+      display: flex;
+      flex-direction: column;
+      flex-wrap: wrap;
+      // width: 100%;
+    }
```

Comment:
Does this need a `flex-wrap`, though? Also, you're defining a `display: flex` twice here which is unnecessary. The desktop styles carry over to mobile styles - you're just tweaking the existing styles so they adapt to a mobile view (in this case, changing the `flex-direction`).